data-raw/sample_tax_bills/sample_tax_bills_summary.R

```diff
@@ -3,9 +3,28 @@ library(stringr)
 library(readr)
 library(data.table)
 
+sum_path <- "data-raw/sample_tax_bills/sample_tax_bills_summary.csv"
+
+
+# add pins/year/class after updating detail
+key_cols <- readr::read_csv(
+  "data-raw/sample_tax_bills/sample_tax_bills_detail.csv"
+) %>%
+  distinct(year, pin, class) %>%
+  mutate(class = as.character(class)) %>%
+  filter(year == max(year))
+
+readr::read_csv(
+  sum_path,
+  col_types = cols(pin = "c", tax_code = "c", class = "c")
+) %>%
+  bind_rows(key_cols) %>%
+  write_csv(sum_path)
```

Comment:
**issue (blocking):** I get the intent but let's not add this programmatically. It's fine if you wanna use it to help jumpstart the manual work on each bill though.

Also, somehow it's mangling the CSV, see below.